src/main/kotlin/com/cognifide/gradle/aem/instance/tail/LogParser.kt

```diff
@@ -2,7 +2,9 @@ package com.cognifide.gradle.aem.instance.tail
 
 import java.io.BufferedReader
 
-class LogParser {
+class LogParser(
+    private val instance: InstanceLoggingInfo = InstanceLoggingInfo.default()
```

Comment:
keep it consistent. use `Log` not `Logging` in naming

Reply:
OK